arangod/RestHandler/RestDocumentHandler.cpp

```diff
@@ -809,6 +820,18 @@ RestStatus RestDocumentHandler::readManyDocuments() {
   opOptions.ignoreRevs =
       _request->parsedValue(StaticStrings::IgnoreRevsString, true);
 
+  // Check if dirty reads are allowed:
+  bool found = false;
+  std::string const& val =
+      _request->header(StaticStrings::AllowDirtyReads, found);
+  if (found && StringUtils::boolean(val)) {
+    opOptions.allowDirtyReads = true;
+    // This will tell `createTransaction` below, that in the case it
+    // actually creates a new transaction (rather than using an existing
+    // one), we want to read from followers. If the transaction is already
+    // there, the flag is ignored.
+  }
```

Comment:
```suggestion
  // This will tell `createTransaction` below, that in the case it
  // actually creates a new transaction (rather than using an existing
  // one), we want to read from followers. If the transaction is already
  // there, the flag is ignored.
  opOptions.allowDirtyReads = _request->parsedValue(StaticStrings::AllowDirtyReads, /*default*/ false);
```